floss/main.py

```diff
@@ -499,6 +502,12 @@ def print_decoding_results(decoded_strings, group_functions, quiet=False, expert
                     print("\nDecoding function at 0x%X (decoded %d strings)" % (fva, len_ds))
                 print_decoded_strings(grouped_strings, quiet=quiet, expert=expert)
     else:
+        if not expert:
+            seen = set()
+            decoded_strings = [x for x in decoded_strings if not (x.s in seen or seen.add(x.s))]
```

Comment:
this is not the most readable implementation, but i appreciate how concise it is :-)

Reply:
I agree, not pretty but still pretty in its own way. There might be a better way to implement this de-keep-the-order-duplication